client-templates/nexus/.env.sample

```diff
@@ -0,0 +1,24 @@
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# The URL to your Nexus Repository Manager
+# If not using basic auth this will only be "<your.nexus.hostname>/repository"
+NEXUS_URL=<username>:<password>@<your.nexus.hostname>/repository
+
+# The URL of the Snyk broker server
+BROKER_SERVER_URL=https://broker.snyk.io
+
+# the fine detail accept rules that allow Snyk to make API requests to your
+# nexus instance
+ACCEPT=accept.json
+
+# The path for the broker's internal healthcheck URL. Must start with a '/'.
+BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# Provide RES_BODY_URL_SUB with the URL of the nexus without credentials and http protocol
+# This URL substitution is required for NPM integration
+# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+
+# Nexus validation url, checked by broker client systemcheck endpoint
+BROKER_CLIENT_VALIDATION_URL=https://$NEXUS_URL/api/system/ping
```

Comment:
is this api path correct?

Reply:
no its not - I think it will be `<username>:<password>@<your.nexus.hostname>/service/rest/v1/status` ?

Reply:
yep `https://$NEXUS_URL/service/rest/v1/status` is shorter way of saying that

Reply:
`NEXUS_URL` is `/repository`...

Reply:
made changes to the file, take a look now and leave comments